modules/ipi-install-configuring-the-raid.adoc

```diff
@@ -6,14 +6,21 @@
 [id="configuring-the-raid_{context}"]
 = Optional: Configuring the RAID
 
-The following procedure configures a redundant array of independent disks (RAID) during the installation process.
+The following procedure configures a redundant array of independent disks (RAID) using baseboard management controllers (BMCs) during the installation process.
 
 [NOTE]
 ====
-. {product-title} supports hardware RAID for baseboard management controllers (BMCs) using the iRMC protocol only. {product-title} {product-version} does not support software RAID.
-. If you want to configure a hardware RAID for the node, verify that the node has a RAID controller.
+If you want to configure a hardware RAID for the node, verify that the node has a supported RAID controller. {product-title} 4.15 does not support software RAID.
```

Comment:
Question: is there a particular reason to not use {product-version} instead of a specific version number that you'd have to update for each release? In either case you'd have to take the time to remove the sentence from `main` once/if it becomes supported.

```suggestion
If you want to configure a hardware RAID for the node, verify that the node has a supported RAID controller. {product-title} {product-version} does not support software RAID.
```